src/module/actor/parsers/available.ts

```diff
@@ -1,41 +1,4 @@
-import {
-  parseNameWTC,
-  parseACWTC,
-  parseAlignmentWTC,
-  parseBiographyWTC,
-  parseConditionImmunitiesWTC,
-  parseDamageImmunitiesWTC,
-  parseDamageResistancesWTC,
-  parseDamageVulnerabilitiesWTC,
-  parseHealthWTC,
-  parseLanguagesWTC,
-  parseMultilineAbilitiesWTC,
-  parseRatingWTC,
-  parseSensesWTC,
-  parseSizeWTC,
-  parseSkillsWTC,
-  parseSpeedWTC,
-  parseAbilitiesWTC,
-  parseTypeWTC,
-  parseVerticalKeyValueAbilitiesWTC,
-  parseItemsWTC,
-} from './wtcTextBlock';
+import { parseActorWTC } from './wtcTextBlock';
+import { parseActorHB } from './homebreweryTextBlock';
 
-export const nameParsers = [parseNameWTC];
-export const ratingParsers = [parseRatingWTC];
-export const typeParsers = [parseTypeWTC];
-export const alignmentParsers = [parseAlignmentWTC];
-export const biographyParsers = [parseBiographyWTC];
-export const languagesParsers = [parseLanguagesWTC];
-export const sizeParsers = [parseSizeWTC];
-export const healthParsers = [parseHealthWTC];
-export const sensesParsers = [parseSensesWTC];
-export const acParsers = [parseACWTC];
-export const damageImmunitiesParsers = [parseDamageImmunitiesWTC];
-export const damageResistancesParsers = [parseDamageResistancesWTC];
-export const conditionImmunitiesParsers = [parseConditionImmunitiesWTC];
-export const damageVulnerabilitiesParsers = [parseDamageVulnerabilitiesWTC];
-export const abilitiesParsers = [parseAbilitiesWTC, parseMultilineAbilitiesWTC, parseVerticalKeyValueAbilitiesWTC];
-export const speedParsers = [parseSpeedWTC];
-export const skillsParsers = [parseSkillsWTC];
-export const itemsParsers = [parseItemsWTC];
+export const actorParsers = [parseActorWTC(), parseActorHB()];
```

Comment:
```suggestion
export const actorParsers = [parseActorWTC, parseActorHB];
```
You need to pass the references to the functions, not the results of their invocations

Reply:
These are actually functions that return an object. Alternatively I could make the objects constants. What do you think?